scripts/configure_pyproject_toml.py

```diff
@@ -122,6 +122,18 @@ def parse_args():
         "scipy-openblas32>=0.3.26",
     ]
 
+    if backend == "lightning_tensor":
+        dependencies += ["cutensor-cu12>=2.0.2"]
```

Comment:
Shall we include this `cutensornet-cu12`?

Reply:
`cutensornet-cu12` and `custatevec-cu12` are currently included as requirements for the Configure Build step (L98/100), should we include them in dependencies too?

Reply:
I have added it for testing currently